pancancer_utilities/tcga_utilities.py

```diff
@@ -120,48 +121,71 @@ def process_y_matrix_cancertype(
 
     return y_df, count_df
 
-def align_matrices(x_file_or_df, y, add_cancertype_covariate=True, algorithm=None):
+def align_matrices(x_file_or_df, y, add_cancertype_covariate=True,
+                   add_mutation_covariate=True):
     """
     Process the x matrix for the given input file and align x and y together
 
     Arguments
     ---------
     x_file_or_df: string location of the x matrix or matrix df itself
     y: pandas DataFrame storing status of corresponding samples
-    algorithm: a string indicating which algorithm to subset the z matrices
+    add_cancertype_covariate: if true, add one-hot encoded cancer type as a covariate
+    add_mutation_covariate: if true, add log10(mutation burden) as a covariate
```

Comment:
If these `add_*` methods keep increasing, you might add a list with functions to generically modify the data.

Reply:
Good idea! I'll keep that in mind.